### DISCUSSION TITLE
Project migration announcement

### DISCUSSION THREAD

**Discussion by sagikazarmark** [bank-vaults/bank-vaults#1878] [Announcements]
Dear Bank-Vaults Community Members, 
  
We hope this message finds you well. We have some important news to share regarding the future of the project. Starting on **April 24th**, we will begin migrating Bank-Vaults from its current home in the _banzaicloud_ GitHub organization to a new, dedicated organization. 
  
This decision has been made after careful consideration, as we believe that this move will bring the project closer to its users and contributors, fostering increased collaboration and innovation. The new organization will provide a more focused environment for the development and growth of Bank-Vaults, ultimately benefiting everyone involved. 
  
Please be assured that this migration will have no impact on existing artifacts, such as container images, and your production deployments. **Your current Bank-Vaults deployments will continue to run smoothly without any interruptions.**
  
Take note of the key details regarding this migration: 
  
1. Timeframe: The migration process will commence on **April 24th** and may take a few weeks to complete, as we ensure all aspects of the project are properly transferred.
2. New Organization: The new organization's name and URL will be announced shortly. We are finalizing all the necessary arrangements and will share the details with the community as soon as they are available.
3. Existing Issues and Pull Requests: All existing issues and pull requests will be migrated to the new repository, ensuring that your contributions are preserved during the transition.
4. Starred Repositories and Forks: If you have starred or forked the Bank-Vaults repository, please remember to do so again in the new organization once the migration is complete.
5. Continuous Integration and Deployment: The CI/CD pipelines will be adjusted accordingly to ensure seamless development and release processes within the new organization.

We understand that this change may require some adjustments on your part, and we apologize for any inconvenience this may cause. However, we firmly believe that this migration is in the best interest of the project and its community. 
  
Please feel free to reach out to us with any questions or concerns you may have regarding this migration. We greatly appreciate your continued support and contributions to the Bank-Vaults project, and we look forward to embarking on this exciting new chapter together. 
  
Thank you and best regards, 
The Bank-Vaults Team

**Comment by leoskyrocker** [comment]
Just for your information as it says:
> Please be assured that this migration will have no impact on existing artifacts, such as container images, and your production deployments.

We did notice that the releases downloads (e.g. https://github.com/banzaicloud/bank-vaults/releases/download/1.15.2/vault-env-darwin-arm64.tar.gz) are no longer available within the old org, but one can simply replace banzaicloud to bank-vaults to get it back.

**Reply by sagikazarmark** [reply]
Thanks for pointing that out @leoskyrocker !

That's an unfortunate result of creating a fork at the original location which we did to make sure previous import paths continue to work.

I'm looking for a way to regenerate previous releases.

Sorry for the inconvenience!

**Reply by LinAnt** [reply]
I had to do some digging to find this Discussion. I opened a PR in the old repository to fix the dead link in the README.md https://github.com/banzaicloud/bank-vaults/pull/1

**Comment by sagikazarmark** [comment]
Please read our follow-up announcement: https://github.com/orgs/bank-vaults/discussions/2118
